### DISCUSSION TITLE
GraphQL Example

### DISCUSSION THREAD

**Discussion by mcritz** [hummingbird-project/hummingbird#77] [Ideas]
I’d like to publish an example integrating GraphQL (with Graphiti) in hummingbird. I’ll be drafting a PR from my fork of the `hummingbird-examples` project.

**Comment by adam-fowler** [comment]
> I’d like to publish an example integrating GraphQL (with Graphiti) in hummingbird. I’ll be drafting a PR from my fork of the `hummingbird-examples` project.

That'd be really cool. Please go ahead.

**Reply by mcritz** [reply]
I just opened PR 10 on hummingbird-examples.
